lib/new_relic/agent/llm/chat_completion_message.rb

```diff
@@ -6,7 +6,7 @@ module NewRelic
   module Agent
     module Llm
       class ChatCompletionMessage < LlmEvent
-        ATTRIBUTES = %i[content role sequence completion_id is_response]
```

Comment:
I'm sure this is said somewhere, but why are removing completion_id? I'm not remembering/ seeing the connection

Reply:
We aren't 🤦 lol. So glad you asked this. I'm not sure what I was thinking. I'll revert.